soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_exclude_overworld.cpp

```diff
@@ -313,11 +313,11 @@ void StaticData::HintTable_Init_Exclude_Overworld() {
                                                  /*german*/ "Man erzählt sich, daß ein #Tauchexperiment# mit #[[1]]# belohnt würde.",
                                                  /*french*/ "Selon moi, l'#expérience de plongée# donne #[[1]]#.", {QM_RED, QM_GREEN}));
                                               // /*spanish*/ Según dicen, #bucear para un experimento# se premia con #[[1]]#.
-    // RANDOTODO: needs translation
+
     hintTextTable[RHT_ZD_FISH] = HintText(CustomMessage("They say that a #fish by a waterfall# hoards #[[1]]#.",
                                              /*german*/ "Man erzählt sich, daß ein #Fisch nahe eines Wasserfalls# #[[1]]# horte.",
-                                             /*french*/ "Selon moi, #[[1]]#.", {QM_RED, QM_GREEN}));
-                                          // /*spanish*/ Según dicen, #[[1]]#.
+                                             /*french*/ "Selon moi, un #poisson près d'une cascade# amasse #[[1]]#.", {QM_RED, QM_GREEN}));
```

Comment:
```suggestion
                                             /*french*/ "Selon moi, un #poisson près d'une cascade# donne #[[1]]#.", {QM_RED, QM_GREEN}));
```

Reply:
My translation is more accurate. Donne = give, amasse = amass, accumulate or hoard.

Reply:
It doesn't make sense in French to use amasse in this case

Reply:
I'll change to a different verb that other hint translations are using in place of "hoard".